sbnanaobj/StandardRecord/SRNuGraphScore.h

```diff
@@ -0,0 +1,42 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRNuGraphScore.h
+////////////////////////////////////////////////////////////////////////
+#ifndef SRNuGraphSCORE_H
+#define SRNuGraphSCORE_H
+
+#include "sbnanaobj/StandardRecord/SRConstants.h"
+
+/**
+ * @brief Categorization of the object/PFP by NuGraph.
+ *
+ * This object summarizes the results from running NuGraph over hits in a slice
+ * (see e.g. [https://sbn-docdb.fnal.gov/cgi-bin/sso/ShowDocument?docid=40585](SBN DocDB 40585).
+ *
+ * The semantic category is the one that most hits belong to.
+ * The fractions describe the categorization of the hits in the object/PFP.
+ */
+namespace caf {
+
+  class SRNuGraphScore {
```

Comment:
The commend needs to be just before the object it's commenting about (in this code, the documentation would have been attached to the `caf` namespace):
```suggestion
namespace caf {

  /**
   * @brief Categorization of the object/PFP by NuGraph.
   *
   * This object summarizes the results from running NuGraph over hits in a slice
   * (see e.g. [https://sbn-docdb.fnal.gov/cgi-bin/sso/ShowDocument?docid=40585](SBN 
   *   DocDB 40585).
   *
   * The semantic category is the one that most hits belong to.
   * The fractions describe the categorization of the hits in the object/PFP.
   */
  class SRNuGraphScore {
```

Reply:
ops...